datastore/blob/azure_storage_api/__init__.py

```diff
@@ -165,14 +165,15 @@ async def is_a_folder(container_client, folder_name):
         raise Exception("Datastore.blob.azure_storage : Unhandled Error")
 
 
-async def create_folder(container_client, folder_uuid=None, folder_name=None, file_name=None):
+async def create_folder(container_client, folder_uuid=None, folder_name=None, blob_name=None):
     """
     creates a folder in the user's container
 
     Parameters:
     - container_client: the container client object to interact with the Azure storage account
     - folder_uuid: the uuid of the folder to be created
     - folder_name: the name of the folder to be created (usually it's uuid)
+    - blob_name: customize blob name to create the folder. Should look like a path for a json file, example : {...}/{...}/{...}.json
```

Comment:
I think we should format the blob_name in this function ourselves.

Since it is a logic depending on our implementation, we wouldn't want to required external devs (BE devs) to use the parameter like this : `blob_name="folder_name/actual_blob_name.json"`. In my opinion, if a parameter is a name, it should only be the plain name, no formatting requiered. The function is here to format it..?  

If we required devs to give the name in a formatted way and we change our logic or implementation for the storage, the BE will also need to update itself to match our logic. Instead, if we format it ourselves, the BE wont need an update and the layers are independent of each other.